examples/kafka/cifar10/cifar10_kafka.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>Probably would be enough to just link to the seldon core installer docs, and maybe pointing out to also set up the seldon core analytics (in case they change they'd be able to install it through there)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>yes</p>